src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToStringTransformer.php

```diff
@@ -118,6 +118,10 @@ public function reverseTransform($value)
             throw new TransformationFailedException('Expected a string.');
         }
 
+        if (true === str_contains($value, "\0")) {
```

Comment:
```suggestion
-        if (true === str_contains($value, "\0")) {
+        if (str_contains($value, "\0")) {
```

Or did i miss something ?

Reply:
@smnandre 

First of all, thank you for the proposal. For the sake of readability and emphasis, it seemed appropriate to me, but I won't have any problem changing it if it violates the project's style guide

Reply:
> if it violates the project's style guide

It does. 😃